src/components/WriteButton/WriteButton.test.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { fireEvent, render, screen } from "@testing-library/react";
+import WriteButton from "./WriteButton";
+
+describe("WriteButton", () => {
+  test("Should be a button", () => {
+    render(<WriteButton />);
+
+    expect(screen.getByRole("button")).toBeInTheDocument();
+  });
+
+  test("Should execute onClick", async () => {
+    let number = 0;
+    const incrementNumber = () => number++;
+
+    render(<WriteButton onClick={incrementNumber} />);
+
+    const button = await screen.findByRole("button");
+
+    fireEvent.click(button);
+    expect(number).toBe(1);
```

Comment:
```suggestion
    const onClickStub = jest.fn()

    render(<WriteButton onClick={onClickStub} />);

    const button = await screen.findByRole("button");

    fireEvent.click(button);
    expect(onClickStub).toBeCalled()
```
I think that approach is more readable